src/screens/Search/FilterSub.js

```diff
@@ -14,7 +14,7 @@ class FilterSub extends Component {
   static navigationOptions = ({ navigation }) => ({
     title: navigation.state.params.title || navigation.state.params.label,
     headerLeft: (
-      <Button transparent color="white" onPress={navigation.goBack}>
+      <Button transparent color="white" onPress={() => navigation.goBack()}>
```

Comment:
`onPress={navigation.goBack}` was it passing `null`?

Reply:
Don't know but when i tried tapping the back button on the view i couldn't go back, so i'm assuming so.